src/server/scripts/Northrend/Naxxramas/boss_gluth.cpp

```diff
@@ -35,21 +38,28 @@ enum Creatures
 
 Position const PosSummon[3] =
 {
-    {3267.9f, -3172.1f, 297.42f, 0.94f},
-    {3253.2f, -3132.3f, 297.42f, 0},
-    {3308.3f, -3185.8f, 297.42f, 1.58f},
+    { 3267.9f, -3172.1f, 297.42f, 0.94f },
+    { 3253.2f, -3132.3f, 297.42f, 0 },
+    { 3308.3f, -3185.8f, 297.42f, 1.58f },
 };
 
 enum Events
 {
-    EVENT_WOUND     = 1,
+    EVENT_WOUND = 1,
     EVENT_ENRAGE,
     EVENT_DECIMATE,
     EVENT_BERSERK,
     EVENT_SUMMON,
+    EVENT_SEARCH_ZOMBIE,
+    EVENT_KILL_ZOMBIE,
 };
 
-#define EMOTE_NEARBY    " spots a nearby zombie to devour!"
+#define EMOTE_NEARBY    "Gluth spots a nearby zombie to devour!"
+#define EMOTE_DECIMATE  "Gluth decimates all nearby flesh!"
+#define EMOTE_ENRAGE    "Gluth becomes enraged!"
```

Comment:
Since the code is being changed it might be a good idea to move these texts to the DB (Talk() and `creature_text`)